### DISCUSSION TITLE
几点新功能建议，虽然不知道作者还会不会更新

### DISCUSSION THREAD

**Discussion by sumeco** [shuax/MouseInc.Settings#461] [Ideas]
1 可以增加个功能自定义热键，直接点击程序任意位置移动程序轻松拖拽窗口。
2可否增加NUMLOCK键自动打开？和capslock一样自动一定时间不用就关闭大写。
3右键管理或者文件重命名管理？

**Comment by wha4up** [comment]
作者退网了
